### ISSUE TITLE
DetaDriveResponse

### ISSUE THREAD

**Issue by DonnC** [yeikel16/deta-dart#14]
@yeikel16 wassup man

Have added the methods implementation but the `DetaDriveResponse`  model is focused on the upload response only, i left `TODOs` on the response of the implementations i added and commented the return statement

Do you have any plans around making it a general drive response model so that all responses are taken into consideration

**Comment by yeikel16** [comment]
> Do you have any plans around making it a general drive response model so that all responses are taken into consideration

Yes, I just noticed and if they change the type of response when a file is deleted and when the file is uploaded by part. They are minimal changes since what varies is only one field, which can be included in the `DetaDriveResponse`.

Response in `Upload Chunked Part`
- Add the field `part` as optional in `DetaDriveResponse`
```json
{
    "name": "file name",
    "upload_id": "a unique upload id"
    "part": 1, // -> this field change
    "project_id": "deta project id",
    "drive_name": "deta drive name"
}
```

Response in `Delete Files`
- Add the field `delete` and `failed` as optional in `DetaDriveResponse`
```json
{
    "deleted": ["file_1", "file_2"] 
    "failed": {
        "file_3": "reason why file could not be deleted",
        "file_4": "reason why file could not be deleted",
    } 
}
```

**Comment by DonnC** [comment]
Makes sense indeed 
Thanks, will adapt that approach
